assertj-core/src/test/java/org/assertj/core/condition/NestableConditionFixtures.java

```diff
@@ -63,6 +63,12 @@ static Condition<Customer> customer(Condition<Customer>... conditions) {
     return nestable("customer", conditions);
   }
 
+  static Condition<ValueCustomer> value(Integer expected) {
+    return verboseCondition(valueCustomer -> expected.equals(valueCustomer.value),
+      "value: " + expected,
+      valueCustomer -> " but was " + valueCustomer.value);
```

Comment:
This seems to duplicate the logic of `assertEquals`. Can it be de-duplicated?

Reply:
Not sure about this, what `assertEquals` method are you referring to?

Reply:
`assertThat(valueCustomer.value).as("value").isEqualTo(expected)`

Reply:
My understanding is that when creating a condition we need a predicate returning true/false, while calling `isEqual` would throw an exception when the condition is not satisfied. 

It would be nice to reuse assertions as conditions. Is there a way of doing that? (a way better than catching the exception an returning false?)